### ISSUE TITLE
term.echo does not respect whitespace

### ISSUE THREAD

**Issue by Kingsy** [jcubic/jquery.terminal#738]
### Issue summary
When using echo() if you feed in a string on multiple lines when each line has whitespace at the beginning, and you use keepWords the resulting output trims the whitespace from the begginning of each line

      let test = `This
                    is
                       a
                          test`
      term.echo(test, { keepWords: true });

Trims whitespace.

      let test = `This
                    is
                       a
                          test`
      term.echo(test);

Echos normally

It is also worth mentioning that this happens on both echo and less.

### Expected behavior

```
This
                    is
                       a
                          test
```

### Actual behavior

Ouput

```
This
is
a
test
```

### Steps to reproduce

      let test = `This
                    is
                       a
                          test`
      term.echo(test, { keepWords: true });

vs

      let test = `This
                    is
                       a
                          test`
      term.echo(test);

### Browser and OS

This does not matter.

**Comment by jcubic** [comment]
This is expected behavior. The whitespace need to be trimmed from the beginning because the original code when wrapped sometimes have single whitespace that was at the end of the line.
```
foo bar baz
```

```
foo bar
 baz
```

This is was the reason for trimming in the code.

What is the use case when you need word wrapping and space in front?
Maybe I can do something with this, the trim was added to quickly fix the space in the front of the line issue, but it probably can be handled without the trim.

**Comment by jcubic** [comment]
Released in version 2.32

**Comment by Kingsy** [comment]
Hi Sorry, I just saw this.

The reason for needing this was I have code blocks inside the echo. Which obviously use whitespace as indentation.

I see a fix went live, what was it?

Thanks and apologies for the delay in the response.

**Comment by jcubic** [comment]
I'm not sure what you mean by "what was it?" the issue was because in the `split_equal` function sometimes there were trailing spaces when keepWords was used, so I've fixed this by using trim, but this also removed spaces in front of the lines which was your problem.

The fix is that it doesn't remove spaces in front.

I just realized that in the original issue I sometimes had one single space in front, I need to check it.
